pages/common/stl2gts.md

```diff
@@ -0,0 +1,24 @@
+# stl2gts
+
+> Convert STL files to GTS (GNU triangulated surface library) format.
+> More information: <https://manned.org/stl2gts>.
+
+- Convert an STL file to a GTS file:
+
+`stl2gts < {{path/to/file.stl}} > {{path/to/file.gts}}`
+
+- Convert an STL file to a GTS file and revert face normals:
+
+`stl2gts --revert < {{path/to/file.stl}} > {{path/to/file.gts}}`
+
+- Convert an STL file to a GTS file and do not merge vertices:
+
+`stl2gts --nomerge < {{path/to/file.stl}} > {{path/to/file.gts}}`
+
+- Convert an STL file to a GTS file and display surface statistics:
+
+`stl2gts --verbose < {{path/to/file.stl}} > {{path/to/file.gts}}`
+
+- Print help for `stl2gts`:
```

Comment:
I like this better but what do you think?
```suggestion
- Display help:
```

Reply:
Can we have a standard, Print should be for when it just outputs it, while Display is for things which can be interacted with, such as man pages.

Reply:
Standard sounds good, however in this case, it just prints it with no possible interaction.

Reply:
I'm inclined towards standards for everything common such as this, like you, and in the past I've proposed many such trivial normalizations, but now I think it might be better to just suggest differenting wording on a PR-by-PR basis, because—

- Not everyone associates the same meaning with the same words,
- If the pages are more identical terms of wording, they seem more machine generated (which isn't what we want),
- Because contributors themselves might have something to say about the usage of verbs,
- A word might be more appropriate for one page but less fit for another, and
- Lots of standards can intimidate contributors and make contribution more cumbersome than it already is.

Reply:
I think it should be fine, its already ok in my opinion, I am ready to merge and resolve this, do you all agree?